docs/build-templates/searx.rst

```diff
@@ -147,7 +148,7 @@ ${fedora_build}
     .. code-block:: sh
 
        # enable debug ..
-       $ sudo -H sed -i -e "s/debug : False/debug : True/g" "$SEARX_SETTINGS_PATH"
+       $ sudo -H sed -i -e "s/'debug : False'/'debug : True'/g" "$SEARX_SETTINGS_PATH"
```

Comment:
Thanks for pointing out this issue! / I guess it is better to fix https://github.com/searx/searx/blob/db703a0283ee169381aeea97c678e666ae508348/utils/searx.sh#L824

or if there is no simple solution, we should quote the all the quotation marks (`"` --> `\"`) in [docs/build-templates/searx.rst](https://raw.githubusercontent.com/searx/searx/master/docs/build-templates/searx.rst)

Reply:
Actually, that pull request is pretty bad, sorry about that.  I hadn't rebased.

I'm making another pull request, hopefully better, right now.

Reply:
There are already double quotes around the -e expression. Otherwise, blank space may be escaped by a few \ .

As you wish, as long as it works correctly.

Reply:
I implemented an alternative approach, see #2277 .. what do you think?

Reply:
I'm having a look at it right now.  Had some trouble getting the PR on my side, and building the documentation.

Reply:
>  Had some trouble getting the PR on my side, and building the documentation.

To fetch and merge a PR from asciimoo/searx, you can do ..

````
  git remote add upstream https://github.com/asciimoo/searx.git
  git pull upstream refs/pull/227/head
````

Alternatively you can fetch and checkout PR's branch from my fork:

````
  git remote add return42 https://github.com/return42/searx.git
  git checkout return42/fix-quotes
````